src/snowflake/cli/api/sql_execution.py

```diff
@@ -279,6 +284,9 @@ def show_specific_object(
         )
         return show_obj_row
 
+    def autocommit(self, value: bool):
```

Comment:
```suggestion
    def set_autocommit(self, value: bool):
```
WDYT? Eventually, maybe we just need `disable_autocommit`?